C/Coin-OR/Clp/build_tarballs.jl

```diff
@@ -68,24 +79,24 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
-    Dependency("Osi_jll", compat="$(Osi_version)"),
-    Dependency("METIS_jll", compat="$(METIS_version)"),
-    Dependency("MUMPS_seq_jll", compat="$(MUMPS_seq_version)"),
-    Dependency("OpenBLAS32_jll", OpenBLAS32_version),
-    Dependency("CompilerSupportLibraries_jll"),
+    Dependency(PackageSpec(name="CoinUtils_jll", uuid="be027038-0da8-5614-b30d-e42594cb92df"), compat="$(CoinUtils_version)"),
```

Comment:
```shell
alexis@HP-Spectre:~/Bureau/git/Yggdrasil$ grep -nr CoinUtils_jll .
./C/Coin-OR/Bonmin/build_tarballs.jl:67:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/Cgl/build_tarballs.jl:63:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/BiCePS/build_tarballs.jl:46:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/ALPS/build_tarballs.jl:47:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/Cbc/build_tarballs.jl:68:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/Osi/build_tarballs.jl:61:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/MibS/build_tarballs.jl:43:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/SYMPHONY/build_tarballs.jl:55:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/CHiPPS_BLIS/build_tarballs.jl:41:    Dependency("CoinUtils_jll", compat="$(CoinUtils_version)"),
./C/Coin-OR/Clp/build_tarballs.jl:82:    Dependency(PackageSpec(name="CoinUtils_jll", uuid="be027038-0da8-5614-b30d-e42594cb92df"), compat="$(CoinUtils_version)"),
```
It will impact a lot of packages if I upgrade `CoinUtils_jll` with `LBT`.
Maybe it's not a good idea to updete the COIN-OR packages.

Reply:
Yes... I should have fixed the compat ages ago, but it wasn't standard practice when I worked on them all. We thought that following semver was enough...

Reply:
See https://github.com/JuliaRegistries/General/pull/100318